src/models/categorygroups.js

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
I see, that some of files with models are named in lowercase. Seems like git didn't track your filename changes.
Use this [page](https://daily-dev-tips.com/posts/git-basics-help-my-case-sensitive-filename-changes-dont-commit/) to correctly rename these files and track them in git